openapi/components/schemas/RevenueEntry.yaml

```
@@ -0,0 +1,47 @@
+type: object
+readOnly: true
+properties:
+  customerId:
+    description: Identifier of the customer which generated the revenue.
```

Awesome! These are much more descriptive.

Now that we are describing them in detail, I think we could shorten them to use `ID` (sorry if I'm giving you mixed messages here )

```suggestion
    description: ID of the customer which generated the revenue.
```